ui/src/pages/job/form/JobFormOthers.js

```diff
@@ -49,7 +51,7 @@ export const JobFormOthers = ({ versions, isSelectVersionDisabled }) => {
           <EuiFlexItem>
             <ModelVersionSelect
               isDisabled={isSelectVersionDisabled}
-              selected={job.version_id}
+              selected={job.version_id.toString()}
```

Comment:
This change was made to prevent some error that's being thrown about the argument `selected` containing an integer instead of a string.